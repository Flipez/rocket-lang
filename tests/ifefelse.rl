i = 0
while (i<3)
  if (i==0)
    puts("if/ef/el: if block")
  ef (i==1)
    puts("if/ef/el: ef block")
  else
    puts("if/ef/el: else block")
  end

  i = i+1
end
