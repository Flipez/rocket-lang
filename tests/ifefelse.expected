"if/ef/el: if block"
"if/ef/el: ef block"
"if/ef/el: else block"
nil
